src/test/java/Homework19.java

```diff
@@ -0,0 +1,42 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+import org.testng.Assert;
+
+import javax.swing.*;
+
+
+public class Homework19 extends BaseTest {
+
+    @Test
+    public void DeletePlaylist() throws InterruptedException {
+        login("dankoyanka@gmail.com", "te$t$tudent");
+
+        WebElement Playlist = driver.findElement(By.xpath
+                ("//li[@class='playlist playlist']"));
+        Playlist.click();
+        Thread.sleep(2000);
+
+        WebElement delbtn = driver.findElement
+                (By.xpath("//button[@class='del btn-delete-playlist']"));
+        delbtn.click();
+        Thread.sleep(2000);
+
+        WebElement ConfirmBtn = driver.findElement
+                (By.xpath("//button[@class='ok']"));
+        ConfirmBtn.click();
+        Thread.sleep(2000);
```

Comment:
you can use wait wait.until(ExpectedConditions.invisibilityOfElementLocated(popUp);

Reply:
This command will be learned in the next lesson

Reply:
do I need to apply the changes and re-submit?

Reply:
no need to resubmit